lredis/cqueues.lua

```diff
@@ -3,6 +3,8 @@ local commands = require "lredis.commands"
 local cs = require "cqueues.socket"
 local cc = require "cqueues.condition"
 local new_fifo = require "fifo"
+local lpeg = require "lpeg"
+local uri_lib = require "lpeg_patterns.uri"
```

Comment:
You're very careful with variable names. I will pay closer attention to this detail.